src/common/TrackedOp.h

```diff
@@ -15,17 +15,20 @@
 #define TRACKEDREQUEST_H_
 #include <sstream>
 #include <stdint.h>
-#include <include/utime.h>
+#include <boost/intrusive/list.hpp>
+#include <atomic>
+
+#include "include/utime.h"
 #include "common/Mutex.h"
 #include "common/histogram.h"
-#include "include/xlist.h"
 #include "msg/Message.h"
 #include "include/memory.h"
 #include "common/RWLock.h"
-#include <atomic>
+
+#define OPTRACKER_PREALLOC_EVENTS 20
```

Comment:
Well it's ops in flight at a time, right? SSD OSDs can handle a lot a more than 10 at a time (at least, I assume).
Dunno how much it actually matters since vector is reasonably intelligent about internal allocations anyway, though.